### DISCUSSION TITLE
How to get values from Index Table?

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#8529] [Q&A] [answered]
How can i get values from one of my index tables?
I need the PrinterContentItemId as string.

![image](https://user-images.githubusercontent.com/61165619/107148586-ade05500-695c-11eb-9af2-7e5187b5b093.png)
![image](https://user-images.githubusercontent.com/61165619/107148602-bfc1f800-695c-11eb-8fa9-fa691fef9d31.png)

**Comment by lampersky** [comment, marked as answer]
You can query index directly, if you don't need any document specific data.

![image](https://user-images.githubusercontent.com/6403130/107150158-b8025380-695c-11eb-97c9-5dc8ee5a2a29.png)

great source to learn about YesSql:
[YesSql Tests](https://github.com/sebastienros/yessql/blob/dev/test/YesSql.Tests/CoreTests.cs)

**Reply by vitalybrandes** [reply]
Same, i always get Null :-(

![image](https://user-images.githubusercontent.com/61165619/107150598-29df9a80-6967-11eb-80b0-d92e6e352537.png)

This is my IndexProvider

![image](https://user-images.githubusercontent.com/61165619/107150611-411e8800-6967-11eb-88d5-d94318f0c304.png)

**Comment by lampersky** [comment]
@vitalybrandes please make sure:
- your migration has created index table (please share your **Migrations.cs**),
- your index table is populated with data,
- you have registered your index provider (Startup.cs).

**Reply by vitalybrandes** [reply]
Thank you! Fix!
